polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -836,6 +840,36 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// TODO write docs
+		#[pallet::call_index(10)]
+		#[pallet::weight(<T as Config>::WeightInfo::establish_system_channel())] // TODO benchmarks
+		pub fn establish_channel_with_system(
+			origin: OriginFor<T>,
+			recipient: ParaId,
+		) -> DispatchResultWithPostInfo {
```

Comment:
Since it's bidirectional, the system chain is not just a recipient. I would change this to either `interlocutor` or `target_system_chain`.